custom_components/esxi_stats/esxi.py

```diff
@@ -132,6 +134,7 @@ def get_host_info(host):
         "name": host_name,
         "state": host_state,
         "version": host_version,
+        "version": host_build,
```

Comment:
Please change the key version to say build so that we don't have a duplicate key.

Reply:
Done ;)